hermetic_build/common/model/generation_config.py

```diff
@@ -105,77 +105,77 @@ def __validate(self) -> None:
                 )
             seen_library_names[library_name] = library.name_pretty
 
+    @staticmethod
+    def from_yaml(path_to_yaml: str) -> "GenerationConfig":
```

Comment:
Are there any logical change in this method?

Reply:
No, only moved so it can be called as `GenerationConfig.from_yaml()`. Provides better readability imo.